### ISSUE TITLE
Recovery phrase verification is difficult

### ISSUE THREAD

**Issue by mhluongo** [iron-fish/node-app#375]
![Screenshot from 2023-06-27 16-21-43](https://github.com/iron-fish/node-app/assets/427505/7ff58ab2-fa03-47b3-8a9a-aec1a2d4b10e)

Instead of requiring the user to write each word, I'd suggest a "checksum-style" alternative. We use this UI for [Taho](github.com/tahowallet/extension), though I'm not sure the license is compatible (GPL-3).

![Screenshot from 2023-06-27 16-25-35](https://github.com/iron-fish/node-app/assets/427505/25bc48be-ea61-4bde-bd37-ee21ea8df3cc)

I'd bet you can combine these two ideas so that...
* People who are writing down recovery phrases have an easier time verifying them, with a very low likelihood that they haven't made a mistake with their phrase
* People who use password managers for recovery phrases can still Command-V the phrase back in to skip the step

**Comment by NullSoldier** [comment]
I agree with this, the requirement to type in the pass phrase is very annoying and unusual for a crypto wallet. The above is much more user friendly.
